packages/nextjs/components/MetaHeader.tsx

```diff
@@ -43,6 +43,20 @@ export const MetaHeader = ({
         <>
           <meta property="og:image" content={imageUrl} />
           <meta name="twitter:image" content={twitterImageUrl} />
+
+          <meta property="fc:frame" content="vNext" />
+          <meta property="fc:frame:image" content={imageUrl} />
+
+          <meta property="fc:frame:button:1" content="🛠️ GitHub" />
+          <meta property="fc:frame:button:1:action" content="link" />
+          <meta property="fc:frame:button:1:target" content="https://github.com/scaffold-eth/scaffold-eth-2" />
+
+          <meta property="fc:frame:button:2" content="📃 Docs" />
+          <meta property="fc:frame:button:2:action" content="link" />
+          <meta property="fc:frame:button:2:target" content="https://docs.scaffoldeth.io/" />
+
+          <meta property="fc:frame:button:3" content="Features →" />
+          <meta property="fc:frame:post_url" content="https://scaffold-eth-frame.vercel.app/api" />
```

Comment:
I guess you are hosting this now?

What about moving it to this repo? So we can have `https://scaffoldeth.io/api`

Reply:
Ooo good idea, got this done. Made a new route under `packages/nextjs/app/api/route.ts`. I had originally used frog.fm but it was giving me issues, so I removed frog.

Reply:
I initially tried just adding to the `pages/api` folder, but the backend code that I have didn't work. I think it's mainly because of the things it needs from `import { NextRequest, NextResponse } from "next/server"`

I tried to make it work for a while but ended up just using the app router and upgrading to next.js 14. I can revert back to 13 and try it again though if you'd like, my backend skills are not very strong